go.mod

```diff
@@ -55,6 +56,30 @@ require (
 	k8s.io/client-go v0.31.1
 )
 
+require (
+	github.com/beorn7/perks v1.0.1 // indirect
+	github.com/cespare/xxhash/v2 v2.3.0 // indirect
+	github.com/cybozu-go/log v1.6.1 // indirect
+	github.com/cybozu-go/netutil v1.4.2 // indirect
+	github.com/cybozu-go/well v1.11.0 // indirect
+	github.com/hashicorp/hcl v1.0.0 // indirect
```

Comment:
License check is failing
```
Not allowed license MPL-2.0 found for library github.com/hashicorp/hcl
```

Reply:
It is in the CNCF exception list (like all the other Hashicorp libraries used by Kubernetes), so using it would be fine:

```json
  {
    "package": "github.com/hashicorp/hcl",
    "license": "MPL-2.0",
    "comment": "not auto-allowlist because: Non-allowlist license(s); approved by GB exception 2019-03-11"
  },
```

From https://github.com/cncf/foundation/blob/1e80c35a4b43868d5070366d9e4501babafd8a52/license-exceptions/cncf-exceptions-2022-04-12.json#L57-L61

I have not been able to figure out[^1] how this makes sense from a legal point of view, that some MPL libraries are acceptable and others aren't.

[^1]: Yes, including asking a TOC member.